src/tailwind.js

```diff
@@ -37,6 +41,9 @@ module.exports = {
       transitionTimingFunction: {
         "out-cubic": "cubic-bezier(0.215, 0.61, 0.355, 1)",
       },
+      transitionProperty: {
+        height: "max-height",
+      },
```

Comment:
actually transition via `max-height` of hardcoded 1000px for the mobile collapse bar is not quite elegant (when more items added, it may overflow) 

i ok to toggle display block instead for now 😬

Reply:
are there anyway to have the animation with display block?